ContactsManagerBackend/ContactsManager/ContactsManagerBL/ContactsManager.cs

```diff
@@ -0,0 +1,18 @@
+﻿using AutoMapper;
+using ContactsManagerDAL;
+using System.Collections.Generic;
+
+namespace ContactsManagerBL
+{
+    public class ContactsManager
+    {
+        private ContactsProvider ContactsProvider { get; } = new ContactsProvider();
+
+        public List<ContactModel> GetContacts()
+        {
+            var contacts = MappingDto.Mapper.Map<List<ContactDto>, List<ContactModel>>(ContactsProvider.GetContacts());
```

Comment:
I'm pretty sure there is an overload that only requires the target type to be specified